extensions-core/google-extensions/src/main/java/org/apache/druid/storage/google/GoogleStorage.java

```diff
@@ -142,14 +147,17 @@ public GoogleStorageObjectMetadata getMetadata(
     );
   }
 
-  public void delete(final String bucket, final String path) throws IOException
+  public void delete(final String bucket, final String path)
   {
+    // Though currently not documented for the GCS delete api, a false response is indicative of file not found.
+    // All other errors appear as StorageException which is a runtime exceptions. Ref:
```

Comment:
Could you add this to the method's javadocs instead?

Also, what's the commitish referring to? It'd be better to point to the version tag `v2.29.1`: https://github.com/googleapis/java-storage/blob/v2.29.1/google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java